controllers/topology/blueprint_test.go

```diff
@@ -248,7 +248,12 @@ func TestGetBlueprint(t *testing.T) {
 			g := NewWithT(t)
 
 			// Set up a cluster using the ClusterClass, if any.
-			cluster := builder.Cluster(metav1.NamespaceDefault, "cluster1").Build()
+			cluster := builder.Cluster(metav1.NamespaceDefault, "cluster1").
+				WithTopology(
+					builder.ClusterTopology().
+						WithClass("class1").
+						Build()).
+				Build()
 			if tt.clusterClass != nil {
```

Comment:
Looks to me like this will be overwritten by l.257-l.263

WDYT about:

```go
			className := "class1"
			if tt.clusterClass != nil {
				className = tt.clusterClass.Name
			}
			cluster := builder.Cluster(metav1.NamespaceDefault, "cluster1").
				WithTopology(
					builder.ClusterTopology().
						WithClass(className).
						Build()).
				Build()

```

Reply:
Simpler again I think we can get rid of the tt.clusterClass != nil check and just have the hardcoded "class1" name across our clusterClasses here.

Reply:
If it fits the data, fine! (I didn't take another look at the tests)